go-controller/pkg/types/resource_status.go

```diff
@@ -1,7 +1,20 @@
 package types
 
+import (
+	"fmt"
+	"strings"
+)
+
 // this file defines error messages that are used to figure out if a resource reconciliation failed
 const (
 	APBRouteErrorMsg         = "failed to apply policy"
```

Comment:
It would make sense for this to follow the same naming: `ErrMsg` vs `ApplyError`